modules/win_ultravnc/templates/ultravnc.ini.epp

```diff
@@ -0,0 +1,71 @@
+[ultravnc]
+passwd=<%= $win_ultravnc::pw_hash %>
+<%# passwd2 is a default value. %>
+passwd2=DE864AF65F98840E2D
```

Comment:
AIUI, passwd2 is the 'read only' password, therefore it should also be looked up from hiera.